### ISSUE TITLE
Training CNN from scratch: Loss function does not decrease

### ISSUE THREAD

**Issue by Goldesel23** [vlfeat/matconvnet#608]
Hi to all, 

So i'm trying to train a CNN from scratch. I have 20 classes, 1000 images per class (750 for training) and the loss function does not decrease even after 500 epochs. I've tried different learning rates, initiatialize the bias to different values, but nothing seems to work.

Anyone might have a clue why the CNN is not learning? 

Thanks in advance for your time.
